python/pyspark/sql/readwriter.py

```diff
@@ -788,7 +788,7 @@ def saveAsTable(self, name, format=None, mode=None, partitionBy=None, **options)
 
     @since(1.4)
     def json(self, path, mode=None, compression=None, dateFormat=None, timestampFormat=None,
-             lineSep=None, encoding=None):
+             lineSep=None, encoding=None, ignoreNullFields=None):
```

Comment:
This seems to be beyond `add doc`. Could you update the PR title, @stczwd ?

Reply:
Okey